src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -910,6 +916,30 @@ static int uct_cuda_copy_md_get_dmabuf_fd(uintptr_t address, size_t length,
     return UCT_DMABUF_FD_INVALID;
 }
 
+static uct_cuda_copy_md_dmabuf_t
+uct_cuda_copy_md_get_dmabuf_internal(const void *address, size_t length,
```

Comment:
maybe unite uct_cuda_copy_md_get_dmabuf_fd and uct_cuda_copy_md_get_dmabuf_internal to a single function - uct_cuda_copy_md_get_dmabuf_fd

Reply:
Combined the functions to a single one.